tests/e2e/velero_helpers.go

```diff
@@ -364,3 +365,37 @@ func verifyVeleroResourceLimits(namespace string, limits corev1.ResourceList) wa
 		return true, nil
 	}
 }
+
+// Create Velero Custom Resource Definition from yaml
+func (v *dpaCustomResource) CreateVeleroFromYaml(yamlTemplatePath string, data interface{}) error {
```

Comment:
Maybe better to use CreateDpaFromYaml or BuildFromYaml as Velero CR changed to DPA CR